src/inputs-controller.ts

```diff
@@ -264,7 +266,7 @@ export class InputsController {
     if (layoutKeyboard) {
       this.selectedDevice[Device.KEYBOARD] = layoutKeyboard.toLowerCase();
     }
-    const handler = globalScene.ui?.handlers[UiMode.SETTINGS_KEYBOARD] as SettingsKeyboardUiHandler;
+    const handler = globalScene.ui?.handlers[UiMode.SETTINGS_KEYBOARD] as KeyboardSettingsUiHandler;
```

Comment:
Q: We should possibly update the `Ui.getHandler<T>()` method to also allow a parameter. Thus we could call
```suggestion
    const handler = globalScene.ui?.getHandler<GamepadSettingsUiHandler>(UiMode.SETTINGS_GAMEPAD);
```
then

Reply:
marking these ones as resolved as well, this should all be addressed in my next PR once this one makes it in